### DISCUSSION TITLE
Show full project

### DISCUSSION THREAD

**Discussion by EricDriussi** [nvim-tree/nvim-tree.lua#1384] [Q&A] [answered]
Hi there!
So I'm sure I'm missing something here, but I can't seem to find it.

If I open a file inside a project from a dir different than the root one (say, the one containing `.git`) and open `nvim-tree`, it will show only the dir containing that file.
Is there a way to make it show everything? Like still keep focus on the current file but just show the whole tree? 

I think it actually behaved like that for me, so I probably screwed up my config -.-U

**Comment by kyazdani42** [comment, marked as answer]
Hi @EricDriussi, i cannot reproduce this behavior, do you have somehow a rooter plugin that might change your base root when opening nvim on a file inside a git repository ?

**Reply by EricDriussi** [reply]
I actually installed a rooter ([notjedi/nvim-rooter.lua](https://github.com/notjedi/nvim-rooter.lua)) after I noticed the described behavior in order to fix it.

Here's what I'm talking about:
No rooter installed:
![without_rooter](https://user-images.githubusercontent.com/46979145/177031158-b463e0f9-645d-43e7-8e8f-3beb21aae29d.gif)

Rooter installed:
![with_rooter](https://user-images.githubusercontent.com/46979145/177031161-06d49bdb-15ba-491c-ae4f-2c8259ecdb03.gif)

Relevant config:
```lua
  disable_netrw = true,

  diagnostics = {
    enable = true,
    icons = {
     ...
    },
  },

  -- Tried messing around with these with no luck :(
  sync_root_with_cwd = true,
  respect_buf_cwd = true,
  update_focused_file = {
    enable = true,
    update_root = true,
  },

  filters = {
    dotfiles = true,
    custom = { '^\\.git' }
  },

  actions = {
    open_file = {
      quit_on_open = false,
      window_picker = {
        enable = true
      }
    }
  },

  view = {
    adaptive_size = true,
    width = 30,
    height = 30,
    hide_root_folder = true,
    side = "left",

    -- Mappings
    mappings = {
      list = {
       ...
      },
    },
  },

  renderer = {
    root_folder_modifier = ":t",
    highlight_git = true,
    icons = {
      ...
      },

      -- Icons
      glyphs = {
        ...
      }
    }
  }
```
Again I wouldn't be surprised if I've set something silly in the config -.-U

**Reply by kyazdani42** [reply]
i'm not sure i understand if this is really an issue with your config, the "without rooter" gif shows opening the file in the same directory, so this behavior is expected, nvim-tree does not have any rooter like behavior.

**Reply by EricDriussi** [reply]
Lovely, thanks a lot!
